pygmt/src/colorbar.py

```diff
@@ -101,6 +103,23 @@ def colorbar(self, **kwargs):
     {panel}
     {perspective}
     {transparency}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # Create a new plot with pygmt.Figure()
+    >>> fig = pygmt.Figure()
+    >>> # Create a basemap figure
+    >>> fig.basemap(region=[0, 10, 0, 3], projection="X10c/3c")
```

Comment:
See https://github.com/GenericMappingTools/pygmt/pull/2417, `frame=True` will be required after PR #2417.
```suggestion
    >>> # Create a basemap
    >>> fig.basemap(region=[0, 10, 0, 3], projection="X10c/3c", frame=True)
```